tests/test_tpa_pipeline.py

```diff
@@ -112,9 +112,8 @@ def test_get_ec_for_request(self, fake_pipeline):
         with mock.patch('enterprise.tpa_pipeline.Registry') as fake_registry:
             fake_registry.get_from_pipeline.return_value = fake_provider
             assert get_enterprise_customer_for_request(request) == self.customer
-        with raises(NotConnectedToOpenEdX) as excinfo:
+        with self.assertRaises(NotConnectedToOpenEdX):
```

Comment:
Leads us to have to (often) maintain explicitly hardcoded error messages in the tests. In fact, I am the one who added the `self.assertIsNotNone(excinfo.value)`, since it was previously a straight up string that was causing issues when error message results changed. That's a waste of time and I don't think there's any difference in detecting the exception vs. seeing if the error message is empty -- I've never run into empty error messages.